Makefile

```diff
@@ -234,6 +234,7 @@ pytest:
 	pytest -m 'local' client/tests/test_ondemand_features.py
 	pytest client/tests/test_localmode_caching.py
 	pytest -m 'local' client/tests/test_resource_registration.py
+	pytest -m 'local' client/tests/test_source_dataframe.py
```

Comment:
for my ocd, can you move client/tests/test_local under (or above) all the pytest -m?